src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  max-width: $calendar-width;
+  gap: $gap-width;
+  padding: 40px $calendar-side-paddings 0;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
```

Comment:
It works perfectly fine with the default `static` value. There is no need rewrite it

Reply:
did't agree, numbers' position on calendars cards break